### PR TITLE
docs(supabase): update comment docs on stream

### PR DESCRIPTION
**Pull request by dshukertjr**
## What kind of change does this PR introduce?

After seeing [this issue](https://github.com/supabase/supabase/issues/14579), I thought it might make sense to add some more details on the comment docs of `.stream()`.

### CODE REVIEW

File: packages/supabase/lib/src/supabase_query_builder.dart [hunk 1]
```diff
@@ -30,18 +30,20 @@ class SupabaseQueryBuilder extends PostgrestQueryBuilder {
           isolate: isolate,
         );
 
-  /// Notifies of data at the queried table
+  /// Returns real-time data from your table as a `Stream`.
   ///
-  /// [primaryKey] list of name of primary key column(s).
+  /// Realtime is disabled by default for new tables. You can turn it on by [managing replication](/docs/guides/realtime/extensions/postgres-changes#replication-setup).
```
Comment (Vinzent03) [inline packages/supabase/lib/src/supabase_query_builder.dart:35]:
```suggestion
  /// Realtime is disabled by default for new tables. You can turn it on by [managing replication](https://supabase.com/docs/guides/realtime/extensions/postgres-changes#replication-setup).
```

**Review by Vinzent03 — CHANGES_REQUESTED** [review]

File: packages/supabase/lib/src/supabase_query_builder.dart (hunk 1 above)
Comment (dshukertjr) [reply packages/supabase/lib/src/supabase_query_builder.dart]: Oh yearh! Thanks for this!

**Review by Vinzent03 — APPROVED** [review]

**Comment by Vinzent03** [comment]
Isn't the type `docs` better  than `fix` here? https://melos.invertase.dev/guides/automated-releases#conventional-commits

**Comment by dshukertjr** [comment]
@Vinzent03 Sure!

**Pull request merged**